tests/Nhs.Appointments.Api.Integration/Scenarios/CreateAvailability/ApplyAvailabilityTemplate.feature

```diff
@@ -1,23 +1,26 @@
 ﻿Feature: Create daily availability
 
-    Scenario: Can apply an availability template
-        Given there is no existing availability
-        When I apply the following availability template
-          | From     | Until        | Days     | TimeFrom | TimeUntil | SlotLength | Capacity | Services |
-          | Tomorrow | Tomorrow_+2 | Relative | 09:00    | 10:00     | 5          | 1        | COVID    |
-        Then the request is successful and the following daily availability is created
-          | Date        | From  | Until | Services | Slot Length | Capacity |
-          | Tomorrow    | 09:00 | 10:00 | COVID    | 5           | 1        |
-          | Tomorrow_+1 | 09:00 | 10:00 | COVID    | 5           | 1        |
-          | Tomorrow_+2 | 09:00 | 10:00 | COVID    | 5           | 1        |
+  Scenario: Can apply an availability template
+    Given there is no existing availability
+    When I apply the following availability template
+      | From     | Until       | Days     | TimeFrom | TimeUntil | SlotLength | Capacity | Services |
+      | Tomorrow | Tomorrow_+2 | Relative | 09:00    | 10:00     | 5          | 1        | COVID    |
+    Then the request is successful and the following daily availability is created
+      | Date        | From  | Until | Services | Slot Length | Capacity |
+      | Tomorrow    | 09:00 | 10:00 | COVID    | 5           | 1        |
+      | Tomorrow_+1 | 09:00 | 10:00 | COVID    | 5           | 1        |
```

Comment:
How much work would it be to have more natural language here such as `2 days from now` instead of Tomorrow_+1